package.json

```diff
@@ -378,5 +378,8 @@
 			"socket.io-client@4.7.2": "patches/socket.io-client@4.7.2.patch",
 			"engine.io-client@6.5.2": "patches/engine.io-client@6.5.2.patch"
 		}
+	},
+	"dependencies": {
```

Comment:
Note: this dep needs to be added to the package.json inside of `eslint-config-fluid`, rather than the repo root. Otherwise the package won't have all its needed deps when installed by other packages.